cmd/conveyor/auth_config.go

```
@@ -61,6 +61,9 @@ func normalizeServerURL(value string) (string, error) {
 	u.Scheme = strings.ToLower(u.Scheme)
 	u.Host = strings.ToLower(u.Host)
 	u.Path = strings.TrimRight(u.Path, "/")
+	if strings.HasSuffix(u.Path, "/mcp") {
```

golangci-lint (2.13.2)

[error] S1017: should replace this if statement with an unconditional strings.TrimSuffix

(staticcheck)